Gems/LmbrCentral/Code/Source/Shape/EditorBaseShapeComponent.cpp

```diff
@@ -172,7 +172,7 @@ namespace LmbrCentral
 
     void EditorBaseShapeComponent::OnShapeColorChanged()
     {
-        m_shapeColor.SetA(AzFramework::ViewportColors::DeselectedColor.GetA());
+        // m_shapeColor.SetA(AzFramework::ViewportColors::DeselectedColor.GetA());
```

Comment:
Is commenting this out possibly debugging stuff?

Reply:
But this will cause shape to be opaque when changing the color of shape